packages/vuetify/src/composables/border-radius.ts

```diff
@@ -1,15 +1,18 @@
 // Utilities
 import { computed } from 'vue'
+import type { Prop } from 'vue'
 import propsFactory from '@/util/propsFactory'
 
 // Types
+export type IRounded = typeof radius[number]
 export interface BorderRadiusProps {
-  rounded?: boolean | number | string
+  rounded?: IRounded
```

Comment:
I'm using `IFoo` because Microsoft's ESLint work recommends using this naming, and I gotten used doing it.